tests/SConscript

```diff
@@ -34,7 +34,6 @@ test_sources = ['main.cc',
                 'unit_tests/pmgd_queries.cc',
                 'unit_tests/helpers.cc',
                 'unit_tests/TDBImage_test.cc',
-                'unit_tests/ImageData_test.cc',
```

Comment:
Better to not use a commit id in the commit message here unless you are done changing code.

Reply:
Agreed, I changed the commit id in 67d2399. Just realized I did the same in 5a1e9dc, so I'll fix that too.